### ISSUE TITLE
Add paragraph classes 1-3 to codebase

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2690]
### Overview
As developers, we use reusable SCSS classes for styling Hack for LA webpages. For this issue, we will add the paragraph classes 1-3 defined in the [Design System](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3689%3A21144) to our codebase.

**Note:** We will not be writing classes for paragraph classes 4-5 because the font size is illegible and should not be used on the website.

<details>
  <summary>Paragraph Classes under Typography in the Design System in Figma</summary>
<img width="862" alt="paragraph-classes-in-design-system" src="https://user-images.githubusercontent.com/31293603/150600536-a2d2aae3-8492-4219-b7b0-6f84839e43f9.png">

</details>

### Action Items
- [x] Locate the Paragraphs section under Typography in the [Design System](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3689%3A21144) in Figma to see how the paragraph classes are defined.
- [x] Look at the following SCSS files to see how title classes were implemented; paragraph classes should be implemented similarly.
  - [x] [_sass/variables/_typography.scss](https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/_sass/variables/_typography.scss)
  - [x] [_sass/elements/_typography.scss](https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/_sass/elements/_typography.scss)
- [x] Based on how each paragraph class is defined in Figma, add SCSS paragraph classes 1-3 to [_sass/elements/_typography.scss](https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/_sass/elements/_typography.scss)
  - [x] You may also need to define some variables in [_sass/variables/_typography.scss](https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/_sass/variables/_typography.scss)
- [x] Update paragraph definitions in Figma (if needed) to match code

#### Merge Team or Technical Leads
Once the pull request associated with this issue is approved and merged, please update the following: 
- [x] Edit issue #2656 by
  - [x] Checking off the dependency for this issue
  - [x] Moving issue #2656 into the New Issue Approval column and removing the Dependency label
- [x] Edit issue #2555 by
  - [x] Checking off the dependency for this issue
  - [x] Moving issue #2555 into the In Progress column, removing the Dependency label, and assigning JessicaLucindaCheng to the issue. (Note: I, Jessica, was working on issue #2555 prior to this one and needed the classes from this issue in order to continue work on issue #2555. So, I had to stop work on issue #2555 and complete this issue first. Once this issue is merged, I want to finish working on #2555 since I have 65% of it done already.) 
  

### Resources/Instructions
- [Design System in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3689%3A21144)
- [_sass/variables/_typography.scss](https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/_sass/variables/_typography.scss)
- [_sass/elements/_typography.scss](https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/_sass/elements/_typography.scss)
- [SASS/SCSS Basics](https://sass-lang.com/guide)
- [SASS/SCSS Documentation](https://sass-lang.com/documentation)

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty @SAUMILDHANKAR   
1) I put Action Items for Technical Leads/Merge Team to release dependency and move the dependent issue to the New Issue Approval column. However, I'm not sure if that's the responsibility of Technical Leads, Merge Team member, or someone else who does those tasks. 
2) Can I be assigned to this too? I need at least two of the paragraph classes for the other issue I'm currently assigned (https://github.com/hackforla/website/issues/2555).

**Comment by JessicaLucindaCheng** [comment]
- Availability: 6 hours this week
- ETA: Tuesday, February 8, 2022

**Comment by JessicaLucindaCheng** [comment]
- Progress: None yet
- Blockers: None
- Availability: 6 hours
- ETA: Tuesday, February 15, 2022

**Comment by JessicaLucindaCheng** [comment]
- Progress: Coding is done. Just need to open a PR for this issue.
- Blockers: I had to work on a different issue because there was a time-sensitive bug. So, I am waiting for that PR to be approved and merged before opening another one, so that I don't have multiple prs open.
- Availability: Will update later. Just waiting for the other pr to be approved and merged.
- ETA: As soon as the other pr is merged and done, will open a pr for this.

**Comment by JessicaLucindaCheng** [comment]
- Progress: Coding is done. Just need to open a PR for this issue.
- Blockers: I had to work on a different issue because there was a time-sensitive bug. So, I am waiting for that PR to be approved and merged before opening another one, so that I don't have multiple prs open.
- Availability: Will update later. Just waiting for the other pr to be approved and merged.
- ETA: As soon as the other pr is merged and done, will open a pr for this.

**Comment by JessicaLucindaCheng** [comment]
- Progress: Coding is done and pr is opened. Fixed a typo as pointed out by Tamara.
- Blockers: none
- Availability: Will update later if further changes are requested
- ETA: Will update later if further changes are requested.
